wavefront_opentracing_sdk/reporting/wavefront.py

```diff
@@ -110,6 +112,8 @@ def set_metrics_reporter(self, wavefront_reporter):
         """
         self._metrics_reporter = wavefront_reporter
         self._registry = wavefront_reporter.registry
+        self._registry.gauge('sender.version', self.CustomGauge(
```

Comment:
let's name it `version` instead of `sender.version` to be consistent with Java SDK naming.